src/ZiggyCreatures.FusionCache/FusionCache.cs

```diff
@@ -904,4 +904,18 @@ internal async ValueTask<(bool error, bool isSame, bool hasUpdated)> TryUpdateMe
 			return (true, false, false);
 		}
 	}
+
+	private void UpdateAdaptiveOptions<TValue>(FusionCacheFactoryExecutionContext<TValue> ctx, ref FusionCacheEntryOptions options, ref DistributedCacheAccessor? dca, ref MemoryCacheAccessor? mca)
+	{
+		// UPDATE ADAPTIVE OPTIONS
+		var maybeNewOptions = ctx.GetOptions();
+		if (ReferenceEquals(options, maybeNewOptions) == false)
+		{
+			options = maybeNewOptions;
+
+			// i'm not sure why we were updating these values before - they are not used after adaptive options are updated
```

Comment:
Yep, [they are](https://github.com/ZiggyCreatures/FusionCache/pull/276/files#diff-e43fd00541c067193a3e4713140acc31de4af7c49fcc3061b74e11da63d974c3L211) (to adaptively save to L1 or L2)